src/dagmc.cpp

```diff
@@ -1,17 +1,35 @@
 #include "openmc/dagmc.h"
 
 #include "openmc/cell.h"
+#include "openmc/constants.h"
 #include "openmc/error.h"
+#include "openmc/file_utils.h"
 #include "openmc/string_utils.h"
 #include "openmc/settings.h"
 #include "openmc/geometry.h"
 
 #include <string>
 #include <sstream>
 #include <algorithm>
+#include <fstream>
+
+namespace openmc {
+
+#ifdef DAGMC
+const bool dagmc_enabled = true;
+#else
+const bool dagmc_enabled = false;
+#endif
+
+}
 
 #ifdef DAGMC
 
+#include "uwuw.hpp"
+#include "dagmcmetadata.hpp"
```

Comment:
I think we only want to include these headers if building with DAGMC. Currently `dagmc.cpp` is always compiled into the OpenMC library. 

Alternatively, I could only include `dagmc.cpp` in the sources list but that might mess with the `dagmc_enabled` variable and its exposure in the `capi`. It seems more clean this way, but I'm happy to look into that. Thoughts?

Reply:
Agreed -- they still need the `#ifdef DAGMC`, but I'd prefer to have them above the definition of `dagmc_enabled` so that all imports come first. Also, just noticed that `DAGMC_FILENAME` needs to be in the `openmc` namespace rather than the global namespace.

Reply:
Oh I see. On it.